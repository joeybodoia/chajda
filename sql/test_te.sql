CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."');
