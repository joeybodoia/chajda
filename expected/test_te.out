CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   పరీక్ష:2 కేసులను:3 రూపొందించడానికి:4 నేను:5 గూగుల్:6 ట్రాన్స్లేట్లో:7 పెడుతున్న:8 నా:9 ఉదాహరణ:10 పరీక్ష:11 వాక్యం:12 ఇది:13
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                             spacy_lemmatize                             
-------------------------------------------------------------------------
   పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్లేట్లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   పరీక్ష:2 కేసులను:3 రూపొందించడానికి:4 నేను:5 గూగుల్:6 ట్రాన్స్లేట్లో:7 పెడుతున్న:8 నా:9 ఉదాహరణ:10 పరీక్ష:11 వాక్యం:12 ఇది:13
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                             spacy_lemmatize                             
-------------------------------------------------------------------------
   పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్లేట్లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                  spacy_lemmatize                                                   
--------------------------------------------------------------------------------------------------------------------
   ":2 పరీక్ష:3 కేసులను:4 రూపొందించడానికి:5 నేను:6 గూగుల్:7 ట్రాన్స్‌లేట్‌లో:8 పెడుతున్న:9 నా:10 ఉదాహరణ:11 పరీక్ష:12 వాక్యం:13 ఇది:14 .:15 ":16
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                spacy_lemmatize                                
-------------------------------------------------------------------------------
   " పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది . "
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                  spacy_lemmatize                                                   
--------------------------------------------------------------------------------------------------------------------
   ":2 పరీక్ష:3 కేసులను:4 రూపొందించడానికి:5 నేను:6 గూగుల్:7 ట్రాన్స్‌లేట్‌లో:8 పెడుతున్న:9 నా:10 ఉదాహరణ:11 పరీక్ష:12 వాక్యం:13 ఇది:14 .:15 ":16
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                spacy_lemmatize                                
-------------------------------------------------------------------------------
   " పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది . "
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   పరీక్ష:2 కేసులను:3 రూపొందించడానికి:4 నేను:5 గూగుల్:6 ట్రాన్స్లేట్లో:7 పెడుతున్న:8 నా:9 ఉదాహరణ:10 పరీక్ష:11 వాక్యం:12 ఇది:13
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                             spacy_lemmatize                             
-------------------------------------------------------------------------
   పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్లేట్లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   పరీక్ష:2 కేసులను:3 రూపొందించడానికి:4 నేను:5 గూగుల్:6 ట్రాన్స్లేట్లో:7 పెడుతున్న:8 నా:9 ఉదాహరణ:10 పరీక్ష:11 వాక్యం:12 ఇది:13
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                             spacy_lemmatize                             
-------------------------------------------------------------------------
   పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్లేట్లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                  spacy_lemmatize                                                   
--------------------------------------------------------------------------------------------------------------------
   ":2 పరీక్ష:3 కేసులను:4 రూపొందించడానికి:5 నేను:6 గూగుల్:7 ట్రాన్స్‌లేట్‌లో:8 పెడుతున్న:9 నా:10 ఉదాహరణ:11 పరీక్ష:12 వాక్యం:13 ఇది:14 .:15 ":16
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                spacy_lemmatize                                
-------------------------------------------------------------------------------
   " పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది . "
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                  spacy_lemmatize                                                   
--------------------------------------------------------------------------------------------------------------------
   ":2 పరీక్ష:3 కేసులను:4 రూపొందించడానికి:5 నేను:6 గూగుల్:7 ట్రాన్స్‌లేట్‌లో:8 పెడుతున్న:9 నా:10 ఉదాహరణ:11 పరీక్ష:12 వాక్యం:13 ఇది:14 .:15 ":16
(1 row)

SELECT spacy_lemmatize('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                spacy_lemmatize                                
-------------------------------------------------------------------------------
   " పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది . "
(1 row)

SELECT spacy_tsvector('te',' "పరీక్ష కేసులను రూపొందించడానికి నేను గూగుల్ ట్రాన్స్‌లేట్‌లో పెడుతున్న నా ఉదాహరణ పరీక్ష వాక్యం ఇది."');
                                                   spacy_tsvector                                                   
--------------------------------------------------------------------------------------------------------------------
 'ఇది':13 'ఉదాహరణ':10 'కేసులను':3 'గూగుల్':6 'ట్రాన్స్లేట్లో':7 'నా':9 'నేను':5 'పరీక్ష':2,11 'పెడుతున్న':8 'రూపొందించడానికి':4 'వాక్యం':12
(1 row)

